include/aspect/material_model/interface.h

```diff
@@ -754,8 +754,13 @@ namespace aspect
                              const FullMatrix<double>      &projection_matrix,
                              const FullMatrix<double>      &expansion_matrix,
                              std::vector<double>           &values_out);
-    }
 
+      /**
+       * Parse an AveragingOperation and alias to an AveragingOperation
+       * that is appropriate for viscosity averaging.
```

Comment:
Please extend the description of the function, I had to read the implementation to understand what is happening here. Something like the following:
```
       * Parse an AveragingOperation and alias to an AveragingOperation
       * that only averages viscosity. In the case of any averaging operation 
       * that averages all properties the corresponding operation for only
       * viscosity is selected (e.g. 'harmonic_average_only_viscosity' instead
       * of 'harmonic_average'). This is useful in places where averaging is
       * performed manually on only the viscosity property.
```